apps/api/src/app/widgets/dtos/session-initialize-request.dto.ts

```diff
@@ -3,14 +3,14 @@ import { IsDefined, IsEmail, IsOptional, IsString } from 'class-validator';
 
 export class SessionInitializeRequestDto {
   @ApiProperty({
-    description: 'Your internal identifire for subscriber',
+    description: 'Your internal identifier for subscriber',
```

Comment:
Sometimes it's hard to tell if things are typos for an old brand name, or just a typo for a word. I guessed wrong on a bunch of these. But, I think I have it right now...